src/main/js/components/subcomponents/Order.js

```diff
@@ -2,4 +2,256 @@
+               response => {
+                   if (!response.ok) {
+                       console.log('Looks like there was a problem. Status Code: ' +
+                           response.status);
+                       return;
+                   }
+
+                   // Examine the text in the response
+                   response.json().then((data) => {
+                       console.log(data._embedded.orders);
+                       this.setState(
+                           {
+                                orders : data._embedded.orders
+                            }
+                        );
+                        console.log("In the render of Orders");
+                        console.log(this.state);
+                        console.log(this.state.orders);
+                        console.log("in render");
+                   });
+               }
+           )
+           .catch(function(err) {
+               console.log('Fetch Error :-S', err);
+           });
+   }
+
+
+   componentDidMount(){
+        this.requestOrders();
+   }
+
+   componentWillUnmount(){
+       console.log("UNMOUNTING UNMOUNTING UNMOUNTING UNMOUNTING UNMOUNTING")
+   }
+
+   handleBackClick(){
+    // this.props.history.goBack();
+    this.props.history.push
+        (
+            {
+                pathname: ('/staff/'),
+                props: this.props
+            } 
+        );
+   }
+}
+
+class StaffOrderList extends React.Component{
+    constructor(props){
+        super(props);
+        
+    }
+
+    render(){
+        const orders = this.props.orders.map(order =>
+            <StaffOrder 
+                key={order._links.self.href}
+                order={order}
+                orderAttributes={this.props.orderAttributes}
+                onUpdate={this.props.onUpdate}
+                //TODO does Staff have authority to delete order?
+
+            />
+        )
+
+        return (
+                <div className="table-container">
+                    <table className="order-table">
+                        <thead>
+                            <tr>
+                                <th scope="col"></th>
+                                <th scope="col">Item</th>
+                                <th scope="col">Price</th>
+                                <th scope="col">Quantity</th>
+                                <th scope="col">Total</th>
+                                <th scope="col">Status</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {orders}
+                        </tbody>
+                    </table>
+                </div>
+        );
+    }
+
+}
+
+export class StaffOrder extends React.Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            price : 4.99,
+            status: this.props.order.status,
+            quantity: this.props.order.quantity,
+            menuItem: {}
+        };
```

Comment:
Confused with the state initializations. What was the reasoning behind `price : 4.99` in line 174